qa/restart-upgrade/src/test/java/org/opensearch/knn/bwc/ModelIT.java

```diff
@@ -140,13 +144,22 @@ public void testKNNModelDefault() throws Exception {
     }
 
     // KNN Delete Model test for model in Training State
+    @Ignore
     public void testDeleteTrainingModel() throws Exception {
-        byte[] testModelBlob = "hello".getBytes(StandardCharsets.UTF_8);
-        ModelMetadata testModelMetadata = getModelMetadata();
-        testModelMetadata.setState(ModelState.TRAINING);
         if (isRunningAgainstOldCluster()) {
+            byte[] testModelBlob = "hello".getBytes(StandardCharsets.UTF_8);
+            ModelMetadata testModelMetadata = getModelMetadata();
+            testModelMetadata.setState(ModelState.TRAINING);
             addModelToSystemIndex(TEST_MODEL_ID_TRAINING, testModelMetadata, testModelBlob);
-        } else {
+
+            Response getResponse = getModel(TEST_MODEL_ID_TRAINING, null);
+            String responseBody = EntityUtils.toString(getResponse.getEntity());
+            assertNotNull(responseBody);
+
+            Map<String, Object> responseMap = createParser(MediaTypeRegistry.getDefaultMediaType().xContent(), responseBody).map();
+            assertEquals(TEST_MODEL_ID_TRAINING, responseMap.get(MODEL_ID));
+            assertEquals(ModelState.TRAINING.toString(), responseMap.get(MODEL_STATE));
```

Comment:
The unit test is for testing delete training model. However, why do we set state and verify the value which doesn't seems related with delete training model.

Reply:
We set the state so that the actual training job doesn't run, the model is just added as immediately in the "training" state.  Verifying the value is not required for the test, and was not removed since the test is currently ignored.